pycsob/client.py

```diff
@@ -233,8 +233,11 @@ def oneclick_init(self, orig_pay_id, order_no, total_amount, currency='CZK', des
             ('totalAmount', total_amount),
             ('currency', currency),
             ('description', description),
+            ('returnUrl', return_url),
+            ('returnMethod', return_method),
+            ('clientInitiated', client_initiated),
```

Comment:
Fun fact: Order of these fields does matter. This order is used to create signature. Wrong order will result in wrong signature.